subsys/trusted_storage/Kconfig

```diff
@@ -133,7 +133,6 @@ config TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK
 	bool "HUK + UID"
 	select HW_UNIQUE_KEY
 	select HW_UNIQUE_KEY_RANDOM
-	select PM_SINGLE_IMAGE
```

Comment:
Actually I think that we need this for the CryptoCell devices since I see that we use PM_ define in this library. So I think that it make sense to avoid selecting this only when Cracen is available or something?